packages/component-library/draft/Kaizen/Table/Table.tsx

```diff
@@ -8,18 +8,24 @@ const sortDescendingIcon = require("@kaizen/component-library/icons/sort-descend
   .default
 
 type TableContainer = React.FunctionComponent
-export const TableContainer: TableContainer = ({ children }) => (
-  <div className={styles.container}>{children}</div>
+export const TableContainer: TableContainer = ({ children, ...otherProps }) => (
+  <div className={styles.container} role="table" {...otherProps}>
+    {children}
+  </div>
 )
 
 type TableHeader = React.FunctionComponent
-export const TableHeader: TableHeader = ({ children }) => (
-  <div className={styles.header}>{children}</div>
+export const TableHeader: TableHeader = ({ children, ...otherProps }) => (
+  <div className={styles.header} role="rowgroup" {...otherProps}>
+    {children}
+  </div>
 )
 
 type TableHeaderRow = React.FunctionComponent
-export const TableHeaderRow: TableHeaderRow = ({ children }) => (
-  <div className={styles.headerRow}>{children}</div>
+export const TableHeaderRow: TableHeaderRow = ({ children, ...otherProps }) => (
+  <div className={styles.headerRow} {...otherProps}>
```

Comment:
Without looking closely at the structure, does this need a ` role="rowheader"` or similar?

Reply:
Yep sure does! I will add that in.